catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -367,6 +368,13 @@ private QueryRequest getRequestWithTagsFilter(QueryRequest request) {
         return newRequest;
     }
 
+    private boolean noDefaultTags(Operation op) {
+        return Optional.ofNullable(op)
+                    .map(Operation::getProperties)
+                    .map(p -> (Boolean) p.getOrDefault("no-default-tags", false))
+                    .orElse(false);
```

Comment:
@pklinef the `"no-default-tags"` should probably be a constant somewhere if this were a for sure fix, but if this is just a temporary thats probably overkill